app/models/null_user.rb

```diff
@@ -0,0 +1,17 @@
+class NullUser
+  def is_admin?
+    false
+  end
+
+  def is_manager?
```

Comment:
Addressed now in an updated version of: 6137adf#diff-3d4dc98d40fd4202d526f609441144dbR2